cmd/crates/soroban-test/tests/it/integration/tx.rs

```diff
@@ -74,12 +74,11 @@ async fn build_simulate_sign_send() {
         .stdout_as_str();
     dbg!("{tx_signed}");
 
-    let output = sandbox
+    sandbox
         .new_assert_cmd("tx")
         .arg("send")
         .write_stdin(tx_signed.as_bytes())
         .assert()
         .success()
-        .stdout_as_str();
-    assert_eq!(output, "SUCCESS");
+        .stdout(predicates::str::contains("SUCCESS"));
```

Comment:
I'm not even sure this works, does SUCCESS get outputted? I added the other assertion as a placeholder and never came back to update it. We should probably make the tx send output more useful than just SUCCESS, such as outputting some print emoji information.

If this works though we can merge.

Reply:
Yeah as is the `send` returns the `GetTransactionResponse` which should definitely include `SUCCESS`, however, I agree that it should do more to check that the transaction had the effect it was supposed to have.